[DOC] Add nestedSet instrinsics to docs (#4898)
grafana/tempo: docs/sources/tempo/traceql/_index.md

{ resource.service.name = "productcatalogservice" } ~ { resource.service.name="frontend" }
```

> You can use nested set intrinsics to request structural details about spans: `nestedSetLeft`, `nestedSetRight`, and `nestedSetParent` columns.

Author: ```suggestion
You can use nested set intrinsics to request structural details about spans.
These intrinsics include `nestedSetLeft`, `nestedSetRight`, and `nestedSetParent.
```

{ resource.service.name = "productcatalogservice" } ~ { resource.service.name="frontend" }
```

You can use nested set intrinsics to request structural details about spans.
These intrinsics include `nestedSetLeft`, `nestedSetRight`, and `nestedSetParent.

> For incomplete traces, all values are set to `0`.

Author: ```suggestion
For incomplete traces, all values are set to `0`. Complete traces have values from `1` to infinity. 
```

Find root spans for complete traces:

```
{ nestedSetLeft=1 }
```

> Select spans of incomplete traces:

Author: ```suggestion
You could use the following example to find and analyze incomplete traces:
```
